crashsimilarity/downloader.py

```diff
@@ -0,0 +1,76 @@
+from datetime import timedelta
```

Comment:
I would prefer to keep the Socorro and Bugzilla code separated in two different modules. The Bugzilla module only contains a single function for now, but could become more complex in the future.